.github/workflows/daily.yaml

```diff
@@ -155,19 +92,10 @@ jobs:
           distribution: 'temurin'
           java-version: ${{ matrix.java }}
           check-latest: true
-          cache: 'maven'
-      - name: Download Maven Repo
-        uses: actions/download-artifact@v4
-        with:
-          name: maven-repo
-          path: .
-      - name: Extract Maven Repo
-        shell: bash
-        run: tar -xzf maven-repo.tgz -C ~
       - name: Build in JVM mode
         shell: bash
         run: |
-          mvn -B --no-transfer-progress -fae -s .github/mvn-settings.xml clean verify
+          mvn -B --no-transfer-progress -fae -s .github/quarkus-snapshots-mvn-settings.xml clean verify
```

Comment:
Because for Linux I just copy there mvn settings https://github.com/quarkus-qe/quarkus-test-suite/blob/ad40c8b0b1ef72c31f20d1e97220d692810964a9/.github/actions/prepare-quarkus-cli/action.yml#L8 it is necessary for Quarkus CLI tests IIRC. I don't remember details TBH, but I think it forced me to do it for same reasons for which we have separate Jenkins jobs for Quarkus CLI tests.

Reply:
ha, here is an explanation https://github.com/quarkus-qe/quarkus-test-suite/blob/ad40c8b0b1ef72c31f20d1e97220d692810964a9/.github/actions/prepare-quarkus-cli/action.yml#L6